docs/developer-guide/porting-2-to-3.md

```diff
@@ -0,0 +1,39 @@
+# Porting Guide: HELICS 2 to 3
+
+Since HELICS 3 is a major version update, there are some breaking changes to the API for developers.
+This guide will try to track what breaking changes are made to the API, and what developers should use
+instead when updating from HELICS 2.x to 3.
+
+## Dependency changes
+
+Support for some older compilers and dependencies have been removed. The new minimum version are:
+
+- C++17 compatible-compiler (minimums: GCC 7.0, Clang 5.0, MSVC 2017, XCode 10, ICC 19)
+- CMake 3.10+
+- ZeroMQ 4.2+
+- Boost 1.65.1+ (if building with Boost enabled)
+
+## Code changes
+
+Changes that will require changing code are listed below based on the interface API used.
+A list of known PRs that made breaking changes is also provided.
+
+### PRs with breaking changes
+
+- https://github.com/GMLC-TDC/HELICS/pull/1363
```

Comment:
```suggestion
- [#1363](https://github.com/GMLC-TDC/HELICS/pull/1363)
```

Reply:
might be worth making a shortcut to this link since it is used in several places